Mapify/SceneInitializers/GameContent/WorldMapSetup.cs

```diff
@@ -17,10 +17,13 @@ public class WorldMapSetup : SceneSetup
         private static bool modifiedMaterial;
         private static Texture defaultTexture;
         public static bool ShowStationNamesOnMap;
+        private static Material mapMaterial;
 
         public override void Run()
         {
-            MapLifeCycle.OnCleanup += () => modifiedMaterial = false;
+            //restore base map texture on exiting a custom map
+            MapLifeCycle.OnCleanup += () => modifiedMaterial = false; mapMaterial.mainTexture = defaultTexture;
```

Comment:
Is it possible for this to get invoked before `defaultMaterial` is initialized?

Reply:
No, I don't think so.